lib/timers/promises.js

```diff
@@ -122,7 +124,110 @@ function setImmediate(value, options = {}) {
   });
 }
 
+function setInterval(after, value, options = {}) {
+  const args = value !== undefined ? [value] : value;
+  if (options == null || typeof options !== 'object') {
```

Comment:
Can `options == null` ever happen here given it has a default value of `{}` ?

Reply:
```js
function f(o = {}) {
  console.log({ o })
}

f(null)
```

Outputs 

`{ o: null }`

This is a copy from `setTimeout`

Reply:
Ah so it can be null but not undefined - you can check `===` in that case probably